testsuite/features/secondary/allcli_config_channel.feature

```diff
@@ -188,63 +188,59 @@ Feature: Management of configuration of all types of clients in a single channel
     And I should see a "-COLOR=red" text
 
 @sle_client
-  Scenario: Check configuration channel and files via XML-RPC for Traditional Client
-    Given I am logged in via XML-RPC configchannel as user "admin" and password "admin"
+  Scenario: Check configuration channel and files via API for traditional client
+    Given I am logged in API as user "admin" and password "admin"
```

Comment:
This feature is all the time login-logout per scenario.

Reply:
Yes, I did not want to change (and break) too many things at once. So you will find scenarios and steps that do individual login-logouts. This is ugly and would need some cleanup work.